docs/managed/access-control.md

```diff
@@ -1,9 +1,9 @@
 # Access Control
 
 PMM Server supports managing access to individual metrics in VictoriaMetrics.  
-[vmgateway](https://docs.victoriametrics.com/vmgateway.html) is placed in front of VictoriaMetrics and helps add extra filters to each query to filter access to metrics, such as `{env=~"prod|staging"}`.
+[vmproxy](https://github.com/percona/pmm/tree/main/vmproxy) is placed in front of VictoriaMetrics and helps add extra filters to each query to filter access to metrics, such as `{env=~"prod|staging"}`.
```

Comment:
vmproxy link seems to be broken atm

Reply:
Could be. I expect the vmproxy PR to be merged soon. We can merge this one afterwards.

Reply:
The PR has now been merged, the link shall work properly.